docs/source/integrations/triton.rst

```diff
@@ -1,3 +1,599 @@
-======
-Triton
-======
\ No newline at end of file
+=======================
+Triton Inference Server
+=======================
+
+*time expected: 12 minutes*
+
+NVIDIA Triton Inference Server [#triton]_ is a high performance, open-source inference server for serving deep learning models.
+It is optimized to deploy models from multiple deep learning frameworks, including TensorRT,
+TensorFlow, ONNX, to various deployments target and cloud providers. Triton is also designed with 
+optimizations to maximize hardware utilization through concurrent model execution and efficient batching strategies.
+
+BentoML now supports running Triton Inference Server through the :ref:`Runner <concepts/runner:Using Runners>`
+architecture. The following integration guide makes the assumption that readers are familiar with BentoML infrastructure.
+Make sure to check out our :ref:`tutorial <tutorial:Creating a Service>` should you wish to learn more about BentoML.
+
+The guide will try to be as comprehensive and detailed as possible, yet we will not be able to cover all the
+features from Triton Inference Server. For more information, please refer to their documentation [#triton_docs]_.
```

Comment:
Maybe avoid "our", "their", and use "BentoML's" and "Triton's" instead for a neutral tone.